.github/workflows/review-code.yaml

```diff
@@ -0,0 +1,61 @@
+name: Review PR Diff
```

Comment:
Lack of context: The comment lacks specific details or context about the action being performed, making it unclear and potentially confusing for other developers. (Severity: Low)